### PR TITLE
Add Brighttalk webinar rss

### LINKED ISSUE #34: Include RSS feed from Brighttalk
Marketing would like to create an RSS feed that sends new webinars from BrightTalk directly to Insights. By doing this, the marketing team will only need to post a new webinar to one site (ie. BrightTalk) and it will automatically appear on Insights as well.

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add Brighttalk webinar rss feed

## QA 

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- Scroll to the bottom of the homepage and look at the 5 webinars from Brighttalk

## Issue

Fixes #34

### CODE REVIEW

**Review by anthonydillon — COMMENTED** [review]
Code looks good :+1:

File: templates/index.html [hunk 1]
```diff
@@ -35,4 +35,28 @@ <h2 class="p-heading--three"><a href="{{ featured_post.relative_link }}">{{ feat
 
   {% include "posts.html" %}
 
+<div class="p-strip is-shallow is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Webinars</h2>
+    {% for webinar in webinars %}
+      <div class="p-media-object" itemscope itemtype="http://schema.org/Event">
+        <div class="p-media-object__details">
```
Comment (anthonydillon) [inline templates/index.html]: Can you use the `thumbnail_url` in the RSS to display the media objects image please.

File: templates/index.html [hunk 2]
```diff
@@ -35,4 +35,28 @@ <h2 class="p-heading--three"><a href="{{ featured_post.relative_link }}">{{ feat
...
+          <h3 class="p-media-object__title" itemprop="name">
+            <a href="{{ webinar.link | safe }}">{{ webinar.title | safe }}</a>
+          </h3>
+          <p class="p-media-object__content">{{ webinar.summary | safe }}</p>
+          <ul class="p-media-object__meta-list">
+            <li class="p-media-object__meta-list-item">
+              <span class="u-off-screen">Topic: </span>{{ webinar.category | safe }}
```
Comment (anthonydillon) [inline templates/index.html]: I think as this has no image it would be worth displaying `Topic:`.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Few comments in the code.

File: templates/index.html (hunk 1 above)
Comment (grahambancroft) [reply templates/index.html]: Done

File: templates/index.html (hunk 2 above)
Comment (grahambancroft) [reply templates/index.html]: Done

**Review by anthonydillon — APPROVED** [review]
Thanks :+1:

**Pull request merged**